checkUserInput.php

```diff
@@ -0,0 +1,23 @@
+<?php
+require_once "functions/allFunctions.php";
+
+$db = connectToDB('pokemon-collection');
+
+$userNickname = $_POST['nickname'];
+$userSpecies = $_POST['species'];
+
+$userGender = $_POST['gender'] ?? NULL;
```

Comment:
You should do some issetting here to make sure some naughty user hasn't directly navigated to this page without filling out the form